core/lib/spree/testing_support/authorization_helpers.rb

```diff
@@ -1,63 +1,21 @@
 module Spree
   module TestingSupport
     module AuthorizationHelpers
-      module CustomAbility
-        def build_ability(&block)
-          block ||= proc{ |u| can :manage, :all }
-          Class.new do
-            include CanCan::Ability
-            define_method(:initialize, block)
-          end
-        end
+      def stub_authorization!(user: Spree.user_class.new, permission_sets: [Spree::PermissionSets::SuperUser])
+        ability = Spree::Ability.new(user)
+        permission_sets.each { |ps| ps.new(ability).activate! }
+        before { allow(Spree::Ability).to receive(:new) { ability } }
```

Comment:
Might be able to get away with stubbing :current_ability on controller here, instead of stubbing new at the ability level. Could cause curious issues depending on the test.